src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+  const filteredStyles = styles.filter(
```

Comment:
The `filter` method is correctly used here to remove empty style declarations. This adheres to the checklist by not mutating the original array.